docs/_static/css/custom.css

```diff
@@ -6,3 +6,18 @@ dl.glossary dt {
 dl.glossary dd {
     margin-top: 2px;
 }
+
+.toggle .header {
+    display: block;
+    clear: both;
+}
+
+.toggle .header:after {
+    content: "▶ show";
+    cursor: pointer;
+}
+
+.toggle .header.open:after {
+    content: "▼ hide";
+    cursor: pointer;
```

Comment:
I guess these are supposed to show? they're not rendering for me, but i've had issues with my CSS rendering in `make livedocs` before. does it reneder on livedoc for you?